src/poker.js

```diff
@@ -1,5 +1,23 @@
 class Poker {
-  // Write your implementation here
+  winningPair(player1, player2) {
+    if (player1[0] !== player1[1] && player2[0] !== player2[1]) {
+      return [];
+    } else if (player1[0] === player1[1] && player2[0] !== player2[1]) {
+      return player1;
+    } else if (player1[0] !== player1[1] && player2[0] === player2[1]) {
+      return player2;
+    } else {
+      if (player1[0] > player2[0]) {
+        return player1;
+      } else {
+        return player2;
+      }
+    }
+  }
 }
-
-module.exports = Poker
+// class Poker {
+//   winningPair(h1, h2) {
+//     return [];
+//   }
+// }
+module.exports = Poker;
```

Comment:
good solution to the core criteria, will be able to use this code, add complexity to expand to the extended criteria